.github/workflows/job-fetch-github-actions-quota.yml

```diff
@@ -0,0 +1,34 @@
+name: Fetch GitHub Actions quota
```

Comment:
If you could stick a robot 🤖 Emoji on the front of this

```suggestion
name: 🤖 Fetch GitHub Actions quota
```

Just helps to identify them in the Actions UI (since we have so many 🙈 )

Reply:
Added.